packages/spacecat-shared-rum-api-client/.eslintignore

```diff
@@ -0,0 +1 @@
+src/common/cruncher.js
```

Comment:
why are we excluding this?

Reply:
no more exclusion. we are inclusive now ❤️